src/win/dtwin.c

```diff
@@ -336,6 +336,40 @@ void dtwin_set_thread_name(DWORD dwThreadID, const char *threadName)
   RaiseException(MS_VC_EXCEPTION, 0, sizeof(info) / sizeof(DWORD), (const ULONG_PTR *)&info);
 }
 
+
```

Comment:
Please add a note about where that function is from, why it's there and under what circumstances we can remove it again. Apart from that it's a necessary evil i suppose.